test/abstract_syntax_tree.coffee

```diff
@@ -91,6 +91,22 @@ test 'Confirm functionality of `deepStrictIncludeExpectedProperties`', ->
   check 'Prop is missing', throws,
     missingProp: {}
 
+# Shorthand helpers for common AST patterns.
+
+EMPTY_BLOCK =
```

Comment:
Added some shorthand helpers to make some of the common AST patterns more concise in the tests